Gemfile

```diff
@@ -88,6 +88,7 @@ gem "pg"
 gem "mini_sql"
 gem "pry-rails", require: false
 gem "pry-byebug", require: false
+gem "pry-stack_explorer", require: false
```

Comment:
Actually should this be added to the `development` and `test` group only? Even though it is not required, it will still be installed.

Reply:
Hmm yeah not sure why all these are up here...I guess they get installed even if they are in the other groups either way -- I will move them down though so it's clearer :+1: